test/JDBC/expected/BABEL-3939-vu-verify.out

```diff
@@ -0,0 +1,113 @@
+exec babel_3939_vu_prepare_p1;
```

Comment:
Please add this into 14_7 and latest schedule files. Please note that 14_7 is expected to be failed until we merge this change into REL2_X_DEV.